admin_guide/pod_network.adoc

```diff
@@ -29,6 +29,22 @@ can now access any pods and services in `<project1>` and vice versa.
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+[[isolating-project-networks]]
+== Isolating Project Networks
+
+To isolate the network of one or more projects from all other projects in the
+cluster:
+
+----
+$ oadm pod-network isolate-projects <project1> <project2>
+----
+
+In the above example, all pods and services in `<project1>` and `<project2>` can
+not access pods and services from other projects in the cluster and vice versa.
```

Comment:
@pravisankar Would it be accurate to say that `<project1>` and `<project2>` would now be on the same pod network and could access each other's pods/services?